et_replay/tools/et_replay.py

```diff
@@ -36,7 +36,7 @@
     TORCH_DTYPES_RNG_str,
 )
 
-from param_bench.et_replay.lib.execution_trace import ExecutionTrace, NodeType
+from et_replay.lib.execution_trace import ExecutionTrace, NodeType
 
 from param_bench.et_replay.lib.utils import trace_handler
```

Comment:
would this also change?

Overall I am curious why we cannot use paths from the root at param_bench?

Reply:
Yes, as we plan to spin off this as a separate tool, I thought it's better to set the path to start from et_replay.